plugins/scaffolder/src/components/MultistepJsonForm/MultistepJsonForm.tsx

```diff
@@ -139,7 +109,7 @@ export const MultistepJsonForm = (props: Props) => {
     if (filteredStep.schema.properties) {
       filteredStep.schema.properties = Object.fromEntries(
         Object.entries(filteredStep.schema.properties).filter(
-          ([key, value]) => {
+          ([key, value]: [string, any]) => {
```

Comment:
can we find a better type than any here?

Reply:
I can also revert it :) As I didn't change that code. 
Intelij IDEA was complaining, decided to satisfy it. 
As I understand schema properties can be anything, do you have any suggestions of defining the better type for value?